src/rust/include/rust/orchard/wallet.h

```diff
@@ -174,10 +174,12 @@ bool orchard_wallet_append_bundle_commitments(
         );
 
 /**
- * Returns the root of the wallet's note commitment tree.
+ * Returns the root of the wallet's note commitment tree by assigning to `root_ret`.
+ * Returns `true`
```

Comment:
```suggestion
Returns the root of the wallet's Orchard note commitment tree with the given number
of confirmations.
```
Also say what conditions could cause a false return.